### PR TITLE
community: add allow_dangerous_requests for OpenAPI toolkits

### PR DESCRIPTION
**Pull request by daviddwlee84**
**OpenAPI allow_dangerous_requests**: community: add allow_dangerous_requests for OpenAPI toolkits

**Description:** a description of the change

Due to BaseRequestsTool changes, we need to pass allow_dangerous_requests manually.

https://github.com/langchain-ai/langchain/blob/b617085af0d14468d5418176d4235229c2c12ffa/libs/community/langchain_community/tools/requests/tool.py#L26-L46

While OpenAPI toolkits didn't pass it in the arguments.

https://github.com/langchain-ai/langchain/blob/b617085af0d14468d5418176d4235229c2c12ffa/libs/community/langchain_community/agent_toolkits/openapi/planner.py#L262-L269 

**Issue:** the issue # it fixes, if applicable

https://github.com/langchain-ai/langchain/issues/19440

If not passing allow_dangerous_requests, it won't be able to do requests.

**Dependencies:** any dependencies required for this change

Not much

### CODE REVIEW

File: libs/community/langchain_community/agent_toolkits/openapi/planner.py
Comment (eyurtsev) [inline libs/community/langchain_community/agent_toolkits/openapi/planner.py:348]: Could you document the variable in args to explain the security concerns of allow dangerous requests?

This will help make sure that folks do not file CVEs against langchain code base

Comment (AdamMomen) [reply libs/community/langchain_community/agent_toolkits/openapi/planner.py:348]: @eyurtsev What's CVE?

Comment (daviddwlee84) [reply libs/community/langchain_community/agent_toolkits/openapi/planner.py:348]: @eyurtsev Sure. I will use the description from `libs/community/langchain_community/tools/requests/tool.py`.
@AdamMomen I think it might mean Common Vulnerabilities and Exposures (CVE).

**Comment by victormimo** [comment]
Anything we can do to help here?

We're waiting for this merge 😬

**Comment by daviddwlee84** [comment]
Waiting for approval here. 0.0

**Comment by victormimo** [comment]
Got it 🙏

**Review by eyurtsev — APPROVED** [review]

**Comment by moshekarmel1** [comment]
Jumping on the thread to push for approval / merge. This is blocking pretty much any use of the OpenAPI toolkit. Thanks!

**Pull request merged**
